mirage/scenarios/registrations.ts

```diff
@@ -128,13 +137,13 @@ export function registrationScenario(
 
     const decaf = server.create('registration', {
         id: 'decaf',
-        title: 'Pending Penguins',
+        title: 'Global Penguins',
```

Comment:
Out of curiosity, why change the name? The previous name indicated that the registration is in a Pending Moderation state, so was descriptive.

Reply:
The original name has now been restored.